<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AzurirajStudent</class>
 <widget class="QDialog" name="AzurirajStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="nadiBtn">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>20</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Pronađi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="jmbagNadi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="azuBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>500</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ažuriraj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ekoTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="infTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="jmbagBr">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="msTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="jmbagLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>JMBAG*</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="ekoLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ekonomija</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="msLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Multimedijski sustavi</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="engTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="matLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Matematika 1</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="imeLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ime</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QComboBox" name="prosaoTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentText">
    <string>DA</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>DA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NE</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="imeTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="prosaoLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Prošao semestar</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="engLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Engleski jezik</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="prezTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="infLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Informatika 1</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="progTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="prezLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Prezime</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="progLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Programiranje 1</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="matTxt">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="infoLbl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
